pcdswidgets/eps_byteindicator.py

```diff
@@ -0,0 +1,155 @@
+from pydm.widgets import PyDMByteIndicator, PyDMChannel
+from qtpy.QtCore import Property, Qt
+from qtpy.QtGui import QColor
+from qtpy.QtWidgets import QTabWidget, QVBoxLayout, QWidget
+
+
+class ByteIndicator_NegativeNums(PyDMByteIndicator):
+    """
+        Modified Byte Indicator calss. Overides update_indicators
+            fucntion to allow for negative numbers to work for ByteIndicator
```

Comment:
Typos and indentation
Python docstrings should be intended to match the triple quote
```suggestion
    Modified Byte Indicator class.
    
    Overrides update_indicators function to allow for negative numbers to work for PyDMByteIndicator.
```